src/core/toga/widgets/optioncontainer.py

```diff
@@ -3,6 +3,63 @@
 from .base import Widget
 
 
+class OptionItem:
+    def __init__(self, container, label, widget, enabled):
+        self._container = container
+        self._index = len(container)
+        self._label = label
+        self._widget = widget
+        self._enabled = enabled
+
+    @property
+    def enabled(self):
+        return self._enabled
+
+    @enabled.setter
+    def enabled(self, value):
+        self._enabled = value
+        self._container.interface._impl.set_option_enabled(self._index, value)
+
+    @property
+    def label(self):
+        return self._label
+
+    @label.setter
+    def label(self, value):
+        self._label = value
+        self._container.interface._impl.set_label(self._index, value)
+
+    def refresh(self):
+        self._widget.refresh()
+
+
+class OptionList:
```

Comment:
For debugging purposes, it may be helpful to override `__repr__` so we can get a good view of what options are in the list, rather than a generic memory dump.